scripts/test/validate-build.ts

```diff
@@ -3,9 +3,10 @@ import { dirname, join, relative } from 'path';
 import { rollup } from 'rollup';
 import ts, { ModuleResolutionKind, ScriptTarget } from 'typescript';
 
-import { NODE_BUILTINS } from '../utils/constants';
-import { BuildOptions, getOptions } from '../utils/options';
-import { PackageData } from '../utils/write-pkg-json';
+import { createNodeLogger, createNodeSys } from '../../src/sys/node/index.js';
```

Comment:
I think this new import is causing CI to fail - would importing just the types fix it?
```suggestion
import type { createNodeLogger, createNodeSys } from '../../src/sys/node/index.js';
```